beacon/entropy_generator.go

```diff
@@ -262,16 +264,21 @@ func (entropyGenerator *EntropyGenerator) trimNextAeons() {
 		if entropyGenerator.lastBlockHeight >= entropyGenerator.nextAeons[0].End {
 			if len(entropyGenerator.nextAeons) == 1 {
 				entropyGenerator.nextAeons = make([]*aeonDetails, 0)
+				changed = true
 			} else {
 				entropyGenerator.nextAeons[0] = nil
 				entropyGenerator.nextAeons = entropyGenerator.nextAeons[1:len(entropyGenerator.nextAeons)]
+				changed = true
 			}
 		} else {
-			// Save aeons to file
-			saveAeons(entropyGenerator.baseConfig.NextEntropyKeyFile(), entropyGenerator.nextAeons...)
 			break
 		}
 	}
+
```

Comment:
Prevents next aeons getting written a lot, making it easier to delete it and restart the node